chia/rpc/wallet_rpc_api.py

```diff
@@ -1727,6 +1727,28 @@ async def nft_set_nft_did(self, request):
         spend_bundle = await nft_wallet.set_nft_did(nft_coin_info, did_id, fee=fee)
         return {"wallet_id": wallet_id, "success": True, "spend_bundle": spend_bundle}
 
+    async def nft_set_bulk_nft_did(self, request):
+        if len(request["nft_coin_list"]) > MAX_NFT_CHUNK_SIZE:
+            return {"success": False, "error": f"You can only set {MAX_NFT_CHUNK_SIZE} NFTs at once"}
+        wallet_id = uint32(request["wallet_id"])
```

Comment:
Would it be possible to assign NFTs from different wallets to a DID? The UI has an aggregated view that shows all NFTs, so I can imagine that users might try to select NFTs across multiple NFT wallets to perform a bulk DID assignment.